maintainers/maintainer-list.nix

```diff
@@ -6751,6 +6751,16 @@
       fingerprint = "A3A3 65AE 16ED A7A0 C29C  88F1 9712 452E 8BE3 372E";
     }];
   };
+  pinpox = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add pinpox``.